pkg/controller/volume/cache/desired_state_of_world_test.go

```diff
@@ -194,6 +194,10 @@ func Test_AddPod_Positive_NewPodNodeExistsVolumeExists(t *testing.T) {
 		t.Fatalf("len(volumesToAttach) Expected: <1> Actual: <%v>", len(volumesToAttach))
 	}
 
+	podScheduled := dsw.GetPods()
+	if len(podScheduled) != 2 {
+		t.Fatalf("len(podScheduled) Expected: <1> Actual: <%v>", len(podScheduled))
```

Comment:
Thanks for pointing it out. fixed.

On Sat, Jun 18, 2016 at 8:37 AM, Peter Zhao <notifications@github.com>
wrote:

> In pkg/controller/volume/cache/desired_state_of_world_test.go
> <https://github.com/kubernetes/kubernetes/pull/27576#discussion_r67602498>
> :
>
> > @@ -194,6 +194,10 @@ func Test_AddPod_Positive_NewPodNodeExistsVolumeExists(t *testing.T) {
> >  		t.Fatalf("len(volumesToAttach) Expected: <1> Actual: <%v>", len(volumesToAttach))
> >  	}
> >
> > +	podScheduled := dsw.GetPods()
> > +	if len(podScheduled) != 2 {
> > +		t.Fatalf("len(podScheduled) Expected: <1> Actual: <%v>", len(podScheduled))
>
> So, the t.Fatalf message should be "len(podScheduled) Expected: <2>
> Actual: <%v>", not "len(podScheduled) Expected: <1> Actual: <%v>"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27576/files/b106cab8833be12bff96236c139d950f5252137b#r67602498>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASSNxRhC87inx5GaSSIb83Qkg7JfjCaEks5qNBCzgaJpZM4I350f>
> .
>



-- 
- Jing